code/edge/src/main/java/com/adobe/marketing/mobile/EdgeConstants.java

```diff
@@ -14,7 +14,7 @@
 final class EdgeConstants {
 
 	static final String EDGE_DATA_STORAGE = "EdgeDataStorage";
-	static final String EXTENSION_VERSION = "1.4.0";
+	static final String EXTENSION_VERSION = "2.0.0";
```

Comment:
As this PR is just part 1 of the changes to adopt Core 2.0, I'm not able to remove any unneeded constants, such as the EdgeType and EdgeSource constants.